package.json

```diff
@@ -5,5 +5,8 @@
   "main": "index.js",
   "repository": "https://github.com/codebase-berkeley/tgif-voting-portal.git",
   "author": "Warren <warrenwwang@berkeley.edu>",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
```

Comment:
likewise with above comment on package.json, can you delete the "dependencies" and "yarn" (lines 9-11)